charts/gardener/controlplane/charts/application/templates/rbac-user.yaml

```diff
@@ -22,6 +22,30 @@ rules:
   - namespaces
   verbs:
   - get
+  - list
+  - watch
```

Comment:
Should we allow all verbs on namespaces like for the other resources?

Reply:
Developers/Admins don't have to be able to manage roles and bindings.

Reply:
Done. With full permissions on RBACs it does not make sense to limit the administrator to only read the namespace resources when he/she can easily create new clusterrole(binding) to get these permissions.